build/gen_stub.php

```diff
@@ -669,6 +818,8 @@ class FuncInfo {
     public $numRequiredArgs;
     /** @var string|null */
     public $cond;
+    /** @var string|null */
+    public $refcount;
```

Comment:
I was hesitant where to put `$refcount`, since it could be added to `ReturnInfo` as well

Reply:
Putting it into ReturnInfo would make sense to me.